### PR TITLE
[Duplicate of #6743] - Integrating MongoDB with Java

### LINKED ISSUE #4206: Integrating MongoDB with Java
#### Proposed title of article
How MongoDB integrate with JAVA 

#### Proposed article introduction
Here, we are going to learn how to connect MongoDB with java. MongoDB is based on a document model and can be written in C++, JavaScript and python.

#### Key takeaways 
After reading the article the reader should have learnt the following:
1. What MongoDB is
2. MongoDB vs Relational database structure
3. Required tools for connection
5. Creating MongoDB collection
6. Selecting a created MongoDB collection
7. What a document is and how it works with different anomalies(insert, update and delete)

#### Article quality
My topic involves basic knowledge that is easy to understand.

#### References
N/A

### PR DESCRIPTION
**Pull request by muthoni2-lucy**
## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. 

**ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**

>As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published. Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This closes #4206

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by muthoni2-lucy** [comment]
hey @srishilesh this is the updated version of pr #6844

**Review by srishilesh — CHANGES_REQUESTED** [review]
Hey @muthoni2-lucy ,

I'm afraid I would have to close this PR again, since it's not up to the mark. You have not addressed all the suggestions yet.

I have pointed out them again in this PR. Please go through them carefully and address them.

Also, `content/articles/index.md` You have not created a directory, instead you have just put your files under `/content/articles` - which is incorrect.

I would highly recommend to take a step back and understand the process at Section. Go through [a few of my PRs](https://github.com/section-engineering-education/engineering-education/pulls?q=is%3Apr+author%3Asrishilesh+is%3Aclosed). See how I have formatted and structured the article. Notice the flow of thoughts. Try to replicate it that way.

Before that understand the whole process of the review, read the reviews carefully.

Ping me in the new PR when done.

Let me know if you have any questions. Feel free to ping other authors to see how they have done. Ask for help.

Thanks.

File: content/articles/index.md [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+``` 
+C:\Program Files\MongoDB\Server\5.0\bin>mongod
+```
+- To run commands on MongoDB, you need to run the mongo command in the other cmd prompt and type the command for MongoDB to open the shell.
+```
+ C:\ Program Files\MongoDB\Server\5.0\bin>mongo
+```
+After you run the code, you can show how many databases are currently available within your MongoDB.
+
+![Available Databases](integrating-mongodb-with-java/available-databases.png).
```
[the remark this answers is not in the document]
Comment (muthoni2-lucy) [reply content/articles/index.md:78]: This are the steps of running the mongodb server, so there is no output

File: content/articles/index.md [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+
+#### Connectivity
+In this tutorial, you'll create a database called SchoolManagementSystems and populate it with a collection called Students. 
+
+Every collection must contain data, and in this tutorial, I'll include the names of two students.
```
[the remark this answers is not in the document]
Comment (muthoni2-lucy) [reply content/articles/index.md:83]: Its an example

[the remark this answers is not in the document]
Comment (muthoni2-lucy) [reply content/articles/index.md:83]: There's a lot you can do its just that i've decided to input names of the students available

File: content/articles/index.md [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+
+Add the following code to your Java class after the created collection.
+```java
+//inserting a new document in the collection which will be student number one.
+Document doc=new Document("name","Christine Muthoni");doc.append("id", "CT201/100091/20");doc.append("course", "Clinical Medicine");doc.append("Gender", "Female");collection.insertOne(doc);System.out.println("Insert is completed");
+
+//insert a new document in the collection, which will be student number two. 
+Document doc2=new Document("name", "Esther Wacera");doc2.append("id", "CT201/10999/14");doc2.append("course", "Civil Engineering");doc2.append("Gender", "Female");collection.insertOne(doc2);System.out.println("Insert is completed");
+```
+In the cmd terminal type
```
[the remark this answers is not in the document]
Comment (muthoni2-lucy) [reply content/articles/index.md:129]: I'm using the both

File: content/articles/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (srishilesh) [reply content/articles/index.md:83]: But, it doesn't look like you are just working with names. That's the reason for this comment.

You are working with student names, IDs and courses.

You must state the remaining as well.

File: content/articles/index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (srishilesh) [reply content/articles/index.md:129]: It looks a bit confusing here.

Fyi, I'm new to this topic. I don't have any idea where to write these codes. Any image would help

**Pull request closed without merging**